bridge_adaptivity/module/tests/test_vpal_engine.py

```diff
@@ -65,12 +65,16 @@ def test_fulfill_payload(self):
     @patch('requests.post', return_value=Mock(status_code=200))
     def test_select_activity(self, mock_post):
         expected_source_url = 'new_activity_source_url'
+        lti_param = "lis_person_sourcedid"
+        launch_params = {lti_param: "test_lis_person_sourcedid"}
+        self.sequence.fulfil_sequence_metadata(self.engine.lti_params, launch_params)
```

Comment:
here's the same about renaming function... It's not clear why `lti_params` is list but `launch_params` is dict

Reply:
launch_params is the attribute of the ToolProvider and returns a dict, our logic require iterable object - generator in our case. All parameters look predictable to me.